examples/agent_otel.yaml

```diff
@@ -0,0 +1,27 @@
+## Minimal configuration to enable Datadog to ship metrics and enable OpenTelemetry.
+
+## The Datadog API key used by your Agent to submit metrics and events to Datadog.
+api_key:
+
+## The environment name where the agent is running. Attached in-app to every
+## metric, event, log, trace, and service check emitted by this Agent.  
+env:
+
+## The site of the Datadog intake to send Agent data to.
+site: datadoghq.com    
+
+## List of host tags. Attached in-app to every metric, event, log, trace, and service check emitted by this Agent.
+tags:
+  - example:changeme
+  - <TAG_KEY>:<TAG_VALUE>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ the other examples have this one line commented out